src/utils/implicits.ts

```diff
@@ -41,9 +41,6 @@ export function registerImportsForImplicit(ast: AST, node: ASTNode, implicit: Im
     case 'bitwise_ptr':
       ast.registerImport(node, ...BITWISE_BUILTIN, [], []);
       break;
-    case 'warp_memory':
-      ast.registerImport(node, ...DICT_ACCESS, [], []);
-      break;
```

Comment:
Why delete this import?

Reply:
Also I think warp memory imports will be handled on plugin side since it is adding initialization and warp memory arguments